common/nym_offline_compact_ecash/src/proofs/proof_withdrawal.rs

```diff
@@ -32,11 +33,15 @@ pub struct WithdrawalReqWitness<'a> {
     pub private_attributes_openings: &'a Vec<Scalar>,
 }
 
-#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, ToSchema)]
 pub struct WithdrawalReqProof {
+    #[schema(value_type = [u64; 4])]
```

Comment:
I think the same applies to Scalars, please double check by trying to send some dummy values across

Reply:
[ditto](https://github.com/nymtech/nym/pull/5235#discussion_r1891517769)